ako-operator/helm/ako-operator/values.yaml

```diff
@@ -76,7 +86,6 @@ ControllerSettings:
   controllerVersion: "18.2.10" # The controller API version
   cloudName: "Default-Cloud" # The configured cloud name on the Avi controller.
   controllerHost: "" # IP address or Hostname of Avi Controller
-  tenantsPerCluster: "false" # If set to true, AKO will map each kubernetes cluster uniquely to a tenant in Avi
```

Comment:
@DixitAakash: Please do not remove existing properties.

Reply:
Added back